src/components/signup/components/Form.js

```diff
@@ -108,4 +107,4 @@ const SignUpForm = ({ form }) => {
 };
 
 const WrappedNormalSignUpForm = Form.create({ name: 'signup' })(SignUpForm);
-export default WrappedNormalSignUpForm;
+export default withRouter(WrappedNormalSignUpForm);
```

Comment:
It makes a DRY and more readable code at App level to handle all routes instead of importing `withRouter`  in modules repeatedly each time one wants to create a route